### DISCUSSION TITLE
Cube example draws a white cube instead of pink

### DISCUSSION THREAD

**Discussion by mittpat** [AryanpurTech/BlueEngine#58] [General]
`examples/shapes/cube.rs` draws a white cube, even though `set_color(1,0,1,1)` is called.
using `set_uniform_color(1,0,1,1)` instead does set the color of the cube to pink.

on ubuntu 22.04.4

thanks for the library, this is fun to use to learn Rust!

**Comment by ElhamAryanpur** [comment]
should be fixed by now
